internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -172,8 +175,9 @@ func Resource() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"id": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: DeprecationMsgOldSchema,
```

Comment:
isn't this one of the fields we said it is not controlled by customers? So would this message popup?

Reply:
Since this is computed, a message pops up if the attribute is being referenced anywhere (see example in PR description). However, the message we have set here doesn't pop-up for computed attributes, instead Terraform gives a generic message `The attribute "id" is deprecated. Refer to the provider documentation for details.`
TF Behavior is very limited in this aspect (computed-only attributes & datasources), maybe this gives more context - https://github.com/hashicorp/terraform/issues/7569